api/billingd/client/client.go

```diff
@@ -139,3 +145,10 @@ func (c *Client) IncCustomerUsage(
 		ProductUsage: productUsage,
 	})
 }
+
+func (c *Client) ReportCustomerUsage(ctx context.Context, key thread.PubKey) error {
```

Comment:
This is only here to be able to test the existence of reported usage, which is usually done daily by the cron reporter.